cms/djangoapps/contentstore/tests/test_libraries.py

```diff
@@ -523,14 +523,6 @@ def test_creation(self):
         self.client.logout()
         self._assert_cannot_create_library(expected_code=302)  # 302 redirect to login expected
 
-        # Now create a non-staff user with no permissions:
-        self._login_as_non_staff_user(logout_first=False)
-        self.assertFalse(CourseCreatorRole().has_user(self.non_staff_user))
-
-        # Now check that logged-in users without any permissions cannot create libraries
-        with patch.dict('django.conf.settings.FEATURES', {'ENABLE_CREATOR_GROUP': True}):
-            self._assert_cannot_create_library()
```

Comment:
@bradenmacdonald I believe removing test is not the best action here. It would be better if the test checked for user without course creation permission to be able to create library.